libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -575,13 +576,15 @@ - (void)beginUserAgentFlow {
                         self.credentials.jwt = nil;
                     }
                 }
+                if (!swapOK) {
```

Comment:
The only other minor thing I'd say is to move the `swapOK` definition [above](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/1676/files#diff-a19763c4fa428fe97ccc907d35e442c3R563) into the `if` block, to reduce its scope.  Extremely minor. 😊